Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM registry.access.redhat.com/ansible-tower-34/ansible-tower
+
+USER root
+
+COPY centos-key /tmp/
+
+ADD https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64 /usr/bin/jq
+ADD https://github.com/mikefarah/yq/releases/download/2.2.1/yq_linux_amd64 /usr/bin/yq
```

Comment:
never heard of yq, presumably jq for yaml?

Reply:
@davidkirwan I hadn't heard of it either until the other day. Its a pre-req package for running the Integreatly installer. See: https://github.com/integr8ly/installation/blob/master/evals/roles/prerequisites/defaults/main.yml#L6-L14